src/pages/Split.tsx

```diff
@@ -98,6 +112,16 @@ const Split: React.FC<Props> = (props) => {
               Add new item
             </IonButton>
           </IonItem>
+          <IonItem>
+            <IonButton
+              disabled={split.all_items.length === 0}
+              expand="block"
+              size="large"
+              onClick={undoLastItem}
+            >
+              Undo last item
+            </IonButton>
```

Comment:
I'm assuming you did this and not a button to delete the expense next to it just because it's easier?

Reply:
didn't even think about removing item by item
that should be easy too